web-main/src/app/state/core/auth/auth.selectors.ts

```diff
@@ -1,14 +1,17 @@
 import * as ngrxStore from '@ngrx/store'
 import { AuthState } from './auth.reducer'
 import * as models from '../../../models'
+import { UserState } from '../user/user.reducer'
```

Comment:
Imports here are wrong, they should import from 
`import * as user from '../user'`